admin_manual/maintenance/migrating.rst

```diff
@@ -62,3 +62,138 @@ When a user tries a URL that is not whitelisted the following error appears:
    :scale: 75%
    :alt: Error message when URL is not whitelisted
 
+=================
+Example migration
+=================
+
+.. note:: For this example to work, you need this on both servers:
+
+* Ubuntu 16.04
+* SSH
+* PermitRootLogin set to "yes"
+* network
+
+optional:
+
+* Domain Name
+* modules: smb-client nfs-common rpcbin
+
+
+Install SSH::
+
+   apt install openssh-server openssh-client -y
+
+Edit SSH-Config::
+
+   nano /etc/ssh/sshd_config
+
+Change PermitRootLogin to yes::
+
+   PermitRootLogin yes
+
+=========
+Migration
+=========
+
+1. Install ownCloud on new server
+
+If you have NAS, then you will need the ```smb-client nfs-common rpcbin``` modules.
+
+2. Put original server in maintenance mode:
+
+Go in owncloud dir::
+
+      cd /var/www/owncloud/
+
+
+Switch to maintenance mode::
+
+      sudo -u www-data php occ maintenance:mode --on
+
+
+wait for 6-7 min and stop apache2::
```

Comment:
exactly. Quote from the doc:
"After waiting for 6-7 minutes for all sync clients to register the server as in maintenance mode, stop the application and/or Web server that serves ownCloud"
https://doc.owncloud.com/server/9.1/admin_manual/maintenance/migrating.html?highlight=migration